Plugins/BenchmarkTool/BenchmarkTool.swift

```diff
@@ -27,13 +27,19 @@ enum Grouping: String, ExpressibleByArgument {
     case benchmark
 }
 
+/// The benchmark data output format.
```

Comment:
I added DocC comments to this OutputFormat enum, which otherwise seemed to be replicated in 3 places. I'm not sure where would be most appropriate, or if DocC additions make any notable difference here.